### PR TITLE
Release 4.0.1

### PR DESCRIPTION
**Pull request by rkewlani**
**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: upgrade_release_version.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+         n)
+          NEWVERSION=${OPTARG};;
+    esac
+done
+
+echo "Old release version is" $OLDVERSION
+echo "New release version is"$NEWVERSION
+
+if [ -z "${OLDVERSION}" ]; then
+    echo "Please specify the 'oldversion' argument";
```
Comment (rikterbeek) [inline upgrade_release_version.sh]: I would not call it old version but current version (with a space in between)

File: upgrade_release_version.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    usage;
+    exit 1;
+fi
+
+if [ -z $NEWVERSION ]; then
+    echo "Please specify the 'newversion' argument";
```
Comment (rikterbeek) [inline upgrade_release_version.sh]: add space between new and version.

File: upgrade_release_version.sh (hunk 1 above)
Comment (rkewlani) [reply upgrade_release_version.sh]: this is argument name so there is no space here.

File: upgrade_release_version.sh (hunk 2 above)
Comment (rkewlani) [reply upgrade_release_version.sh]: same here. it is argument name.

File: upgrade_release_version.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        o)
+          OLDVERSION=${OPTARG};;
+
+         n)
+          NEWVERSION=${OPTARG};;
+    esac
+done
+
+echo "Old release version is" $OLDVERSION
+echo "New release version is"$NEWVERSION
```
Comment (Aleffio) [inline upgrade_release_version.sh]:
```suggestion
echo "New release version is" $NEWVERSION
```

**Pull request merged**